openaddr/conform.py

```diff
@@ -154,7 +154,7 @@ def is_in(path, names):
     if path.lower() in names:
```

Comment:
I think these functions will be called frequently (once per row of source data), so we should compile these regexes once rather than for every function call.